install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -318,8 +318,13 @@ exported volume on the NFS server should conform to the following conditions:
 - Each export must be:
 +
 ----
-/<example_fs> *(rw,root_squash)
+/<example_fs> *(rw,root_squash,sync,no_wdelay)
 ----
++
+Where the `sync` option causes server to commit all the changes to the
+underlying storage before sending a reply. And `no_wdelay` stops the server
+from delaying the writes. They proved to be necessary for correct functioning
```

Comment:
On 04/15/2016 02:43 PM, Paul Weil wrote:
> In install_config/persistent_storage/persistent_storage_nfs.adoc <https://github.com/openshift/openshift-docs/pull/1908#discussion_r59921244>:
> 
>>  ----
>> ++
>> +Where the `sync` option causes server to commit all the changes to the
>> +underlying storage before sending a reply. And `no_wdelay` stops the server
>> +from delaying the writes. They proved to be necessary for correct functioning
> 
> remove the "They..." sentence.
I don't understand why the `sync` is necessary... Doesn't just `no_wdelay` work?

steved.